docs/spec/executable/tes-http-serverd.md

```diff
@@ -6,39 +6,26 @@ credentials for other processes to acquire.
 For information on TES, see
 <https://docs.aws.amazon.com/greengrass/v2/developerguide/interact-with-aws-services.html>.
 
-The following is the current daemon implementation flow:
+The following are the requirements for the `tes-http-serverd`:
 
 - [tes-http-serverd-1] The `tes-http-serverd` daemon will use the `tes` daemon
-  for acquiring credentials.
-- [tes-http-serverd-2] On connection, the `tes-http-serverd` daemon will
-  validate the provided token and establish an http connection with the server.
-- [tes-http-serverd-3] On request, the `tes-http-serverd` daemon will validate
+  for acquiring TES credentials.
+- [tes-http-serverd-2] On request, the `tes-http-serverd` daemon will validate
   the request and headers.
-- [tes-http-serverd-4] On request, the `tes-http-serverd` daemon will make a
+- [tes-http-serverd-3] On request, the `tes-http-serverd` daemon will make a
   request to the `tes` daemon to acquire the temporary credentials vended, and
   return to the caller.
+- [tes-http-serverd-4] There are no command line arguments.
```

Comment:
Do we need to specify this? Unless we specify that there are command line arguments, there should not be any.